if __name__ == '__main__':
    try:
        contador = o

        divisao = 4 / contador

        print('TESTE')
    except Exception as e:
        print('Ocorreu um erro, favor tente novamente!')
    
    print('TESTE2')